dashboard/src/shared/AxiosInstance.ts

```diff
@@ -25,15 +25,25 @@ export function createAxiosInterceptors(axios: AxiosInstance, store: Store<IStor
   axios.interceptors.response.use(
     response => response,
     e => {
+      const dispatch = store.dispatch as ThunkDispatch<IStoreState, null, Action>;
       const err: AxiosError = e;
+      if (
+        err.config.xsrfCookieName === "XSRF-TOKEN" &&
```

Comment:
this doesn't seem relevant to catching this error to me, I'm guessing axios just sets this as a default